db/seeds/development/rooms_seed.rb

```diff
@@ -0,0 +1,6 @@
+(1..25).each do |index|
+  new_room = Room.create(
```

Comment:
We should also assign the `room` to a `conference` right?
Let's just seed all rooms to `Conference.first`. Other seeds should have examples!